wellcomeml/ml/keras_vectorizer.py

```diff
@@ -47,6 +47,12 @@ def decode(self, encoded_text):
             return self.tokenizer.sequences_to_texts([encoded_text])[0]
         return self.tokenizer.sequences_to_texts(encoded_text)
 
+    def tokenize(self, text):
```

Comment:
Is that correct? Is that really the type of tokenisation we wanna do? (Just split by spaces?)

It is slightly different from transformers (when you call `.encode` and get the tokens), which I think also maybe excludes some characters etc...

Reply:
We want to expose the tokenisation that Keras uses see https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/text/Tokenizer. They seem to split by whitespace `' '`.

In transformers we similarly expose the `tokenize` method they have but in keras there is no method so we are just recreating it.